test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Content.Tests.ps1

```diff
@@ -6,16 +6,16 @@ function Get-RandomString
 {
     param ( [int]$length = 8 )
```

Comment:
For consistency, perhaps we should use Pascal casing here as well: $Length so that the subsequent use below has consistent casing

Reply:
Fixed.